bugbug/model.py

```diff
@@ -418,6 +430,17 @@ def train(self, importance_cutoff=0.15):
 
             if not is_multilabel:
                 y_pred_filter = self.le.inverse_transform(y_pred_filter)
+                y_pred_filter = np.asarray(
+                    np.concatenate(
+                        (
+                            y_pred_filter,
+                            np.array(["NOT_CLASSIFIED"] * len(not_classified_indices)),
```

Comment:
Use "___NOT_CLASSIFIED___" so it's clearer it's an internal name (no class is ever going to be called like this).

Reply:
Note: with \_\_\_ before and after NOT_CLASSIFIED